rotkehlchen/api/rest.py

```diff
@@ -1314,9 +1314,9 @@ def user_premium_key_remove(self) -> Response:
         # else
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
-    def query_all_assets(self, filter_query: AssetsFilterQuery) -> Response:
+    def query_all_assets(self, filter_query: 'AssetsFilterQuery') -> Response:
```

Comment:
why are you quoting it in string here? It's imported in the file so there is no reason for it